example_configs/home-assistant.md

```diff
@@ -16,9 +16,20 @@ homeassistant:
     - type: homeassistant
     - type: command_line
       command: /config/lldap-ha-auth.sh
-      # Only allow users in the 'homeassistant_user' group to login.
-      # Change to ["https://lldap.example.com"] to allow all users
-      args: ["https://lldap.example.com", "homeassistant_user"]
+      # arguments: [<LDAP Host>, <regular user group>, <admin user group>, <local user group>]
+      # <regular user group>: Only allow users in the <regular user group> (e.g., 'homeassistant_user')
+      # group to login. Users will have the default 'system-users' permission.
+      #
+      # <admin user group>: Allow users in the <regular user group> (e.g., 'homeassistant_user') with
```

Comment:
allow users belonging to both the `<regular user group>` and `<admin user group>`

WDYT?

Reply:
My expression was inaccurate. `<regular user group>` is a list of users that are permitted to access `homeassistant`. Upon that, `<admin user group>` enables the users within `<regular user group>` to be in the admin group.